services/src/unit-test/java/uk/gov/moj/sdt/services/utils/MessagingUtilityTest.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.moj.sdt.services.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.domain.api.IBulkSubmission;
+import uk.gov.moj.sdt.domain.api.IIndividualRequest;
+import uk.gov.moj.sdt.domain.api.ITargetApplication;
+import uk.gov.moj.sdt.services.messaging.MessageWriter;
+import uk.gov.moj.sdt.services.messaging.SdtMessage;
+import uk.gov.moj.sdt.services.messaging.api.IMessageWriter;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+import uk.gov.moj.sdt.utils.transaction.synchronizer.MessageSynchronizer;
+import uk.gov.moj.sdt.utils.transaction.synchronizer.api.IMessageSynchronizer;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class MessagingUtilityTest extends AbstractSdtUnitTestBase {
+
+    @BeforeEach
+    @Override
+    public void setUp() {
+    }
+
+    @Test
+    public void testEnqueueRequestNew() {
+        // Given
+        IIndividualRequest individualRequest = mock(IIndividualRequest.class);
+        IBulkSubmission bulkSubmission = mock(IBulkSubmission.class);
+        ITargetApplication targetApplication = mock(ITargetApplication.class);
+        IMessageSynchronizer messageSynchronizer = mock(IMessageSynchronizer.class);
+        IMessageWriter messageWriter = mock(IMessageWriter.class);
```

Comment:
These are used to set up the MessagingUtility object in each test.  Might as well create class fields for them, annotate them with Mock and set up a MessagingUtility class field in the setUp() method.

Reply:
done